lib/src/base-styles/_nx-typography.scss

```diff
@@ -65,7 +65,7 @@
 
   color: $nx-heading-color;
   font-size: $nx-font-size-heading-level-3;
-  margin-bottom: $nx-spacing-xs;
+  margin-bottom: $nx-spacing-sm;
```

Comment:
This seems to be the reason for https://github.com/sonatype/sonatype-react-shared-components/pull/129#issuecomment-688931254

Reply:
> This seems to be the reason for [#129 (comment)](https://github.com/sonatype/sonatype-react-shared-components/pull/129#issuecomment-688931254)

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/commit/fe5d9bf5020868de4de1d8ea122982783c885d85